integration/spark/src/test/scala/org/apache/spark/carbondata/restructure/vectorreader/AlterTableColumnRenameTestCase.scala

```diff
@@ -38,6 +38,15 @@ class AlterTableColumnRenameTestCase extends QueryTest with BeforeAndAfterAll {
     assert(null == carbonTable.getColumnByName("empname"))
   }
 
+  test("CARBONDATA-4053 test rename column when column name is a") {
+    sql("create table simple_table(a int) stored as carbondata")
+    sql("alter table simple_table change a a1 int")
+    val carbonTable = CarbonMetadata.getInstance().getCarbonTable("default", "simple_table")
```

Comment:
This issue happens when you have table properties say SORT_COLUMNS = "a,b"
Then the column name changed from a to a1, then it was changing SORT_COLUMNS = "a1" instead of SORT_COLUMNS = "a1,b". So, your current testcase cannot reproduce issue. so, add the testcase that gives issue without the change by keeping some table properties.

Reply:
Done, modify the testcase, now can cover the scenario you mentioned.